src/nistEncode.test.ts

```diff
@@ -526,6 +528,80 @@ describe('positive test:', () => {
     expect((result as Success<Buffer>).value).toMatchSnapshot();
     // await fsPromises.writeFile('type-1-2-13.tdf', (result as Success<Buffer>).value);
   });
+
+  it('Type-1, Type-2, Type-9, Type-13 with default options - encode into a Buffer', () => {
+    const nist: NistFile = {
+      1: {
+        2: '0201',
+        3: [
+          ['1', '3'],
+          ['2', '00'],
+          ['9', '1'],
+          ['13', '1'],
+        ],
+        4: 'LFFS',
+        5: '20200122',
+        7: 'WVNGI0520',
+        8: 'WVNGI0520',
+        9: 'UCNRHMPXFAPV',
+        11: '39.37',
+        12: '39.37',
+      },
+      2: {
+        6: 'FBI LATENT PRINT',
+        10: [['UCN: RHMPXFAPV', 'BSI: 2000167243433']],
+        11: '1111',
+        74: '00',
+        79: '20',
+        83: 'Y',
+        95: 'Y',
+        98: '1',
+      },
+      9: [
+        {
+          3: '7',
+          4: 'U',
+          300: [['1971', '2873', '0', '0', '0,0-0,2873-1971,2873-1971,0']],
+          301: [['0', '30']],
+          302: ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10'],
```

Comment:
Here's what I found in a documentation of nist format called "NIST Special Publication 500-245", chapter 16.2.6 Field 9.006: Finger position (FGP) (page 28)

> If the exact finger or palm position cannot be determined, multiple finger positions may be entered, separated by the “RS” character.

This is the case if you look at the content of the file.

Reply:
Hello Ikhsan, NIST SP 500-245 from September 2000 is a great source of information. Unfortunately this revision does not provide any information about field 9.302, which my comment targeted. The first mention of field 9.302 is in NIST SP 500-290 from 2013.

In this specification, Table 30 gives some guidance on the content of field 9.302:
"Subfields: Repeating sets of information items".
In other words, 9.302 in general consists of the following information items which repeat:
```
[[FGP1, FSM1, OCF1, SGP1], [FGP2, FSM2, OCF2, SFP2], [FGP3, FSM3, OCF3, SFP3],...]
  or with NIST separators:
FGP1<US>FSM1<US>OCF1<US>SGP1<RS>FGP2<US>FSM2<US>OCF2<US>SFP2<RS>FGP3<US>FSM3<US>OCF3>US>SFP3
```
Now the specification precises that only FGP is mandatory; FSM, OCF and SGP are optional.
So if only FGP is present and FSM+OCF+SGP are omitted, also the following notation is valid:
```
[[FGP1], [FGP2], [FGP3],...]
FGP1<RS>FGP2<RS>FGP3
```

Recall again the notation explained in https://github.com/ivosh/node-nist/blob/master/src/index.ts#L30. There is a big difference between:
```
[[1], [2], [3]]
and
[1, 2, 3]
```

I hope the matter has been clarified.

Reply:
Not really a bug in `nistDecode`. Rather a missing feature to correctly decode Type 9 fields with such a semantics.
Recall that typically a field with <RS> gets decoded to a simple array `[1, 2, 3...]`. However in this special case, based on the semantics of this particular field, it should decode to array `[[1], [2], [3]...]` as if there were also a <US> present. You can see that in [decodeNistSubfield](https://github.com/ivosh/node-nist/blob/master/src/nistDecode.ts#L102).

I found the following fields (for the supported NIST record types) where only one information item of a set is mandatory and the rest is optional:
* 9.135
* 9.302
* 9.324
* 9.354
* 9.355
* 9.356
* 10.042
* 10.048
* 10.995
* 10.997
* 13.995
* 13.997
* 14.995
* 14.997

This PR should correctly handle the Type 9 fields because it's called 'Add support for NIST record Type-9'. I will handle the other types in a separate PR.  Could you please extend `nistDecode` to add an additional behaviour based on the field number as above for Type-9 record?

Reply:
I see, this missing feature is better to be handled in another PR.

Concerning your request:
> Could you please extend nistDecode to add an additional behaviour based on the field number as above for Type-9 record?

I'm sorry, I don't get this part. Do you want me to to add more fields in the interface `NistType9Record` ? Or do you want me to add a function (called `decodeType9Record`) in `nistDecode.ts` that handles Type-9 record ?

Reply:
> I see, this missing feature is better to be handled in another PR.

Exactly.

Let's bound this PR to add a basic support for Type-9 record. This has been done, the record can be decoded and encoded. The work on the correct representation of the fields listed in the previous comment can continue in another PR.